pom.xml

```diff
@@ -603,7 +614,12 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.18.1</version>
         <configuration>
-          <argLine>-Duser.language=en -Duser.country=US -XX:-UseSplitVerifier</argLine>
+          <systemPropertyVariables>
+            <jacoco-agent.destfile>${basedir}/target/jacoco.exec</jacoco-agent.destfile>
+          </systemPropertyVariables>
+          <properties>
+            <argLine> -Duser.language=en -Duser.country=US -XX:-UseSplitVerifier</argLine>
+          </properties>
```

Comment:
@Godin 
According to JaCoCo's documentation:
>If your project already uses the argLine to configure the surefire-maven-plugin, be sure that argLine defined as a property, rather than as part of the plugin configuration.

But it is for prepare-agent, and now we use offline instrumentation, so, we do not need properties)